net/udpbroadcastrelay/src/opnsense/mvc/app/controllers/OPNsense/UDPBroadcastRelay/forms/dialogEdit.xml

```diff
@@ -43,6 +43,31 @@
       <type>checkbox</type>
       <help><![CDATA[Use the ID to set the TTL Value of the packet (Original method) rather than DSCP, default is not ticked]]></help>
    </field>
+   <field>
+      <id>udpbroadcastrelay.allowcidr</id>
+      <label>Allowed CIDRs</label>
+      <style>tokenize</style>
+      <type>select_multiple</type>
+      <allownew>true</allownew>
+      <help><![CDATA[Only allow packets from a range of IP source addresses]]></help>
```

Comment:
```suggestion
      <help>Only allow packets from a range of IP source addresses.</help>
```

adhere to style at least for new submissions. would be nice if you could submit cleanups for the others.